apps/bfd-server/bfd-server-war/src/main/config/server-start.sh

```diff
@@ -139,7 +139,8 @@ if [[ "${cygwin}" = true ]]; then keyStore=$(cygpath --mixed "${keyStore}"); fi
 if [[ "${cygwin}" = true ]]; then trustStore=$(cygpath --mixed "${trustStore}"); fi
 
 # Read the server port to be used from the ports file.
-serverPortHttps=${BFD_PORT:-$(grep "^server.port.https=" "${serverPortsFile}" | tr -d '\r' | cut -d'=' -f2)}
+serverPortHttps=$(grep "^server.port.https=" "${serverPortsFile}" | tr -d '\r' | cut -d'=' -f2)
```

Comment:
**This** is a fun one.  I spent way too long wondering why this couldn't obtain the proper port written to file during the maven build.  As it turns out, `BFD_PORT` is an existing environment variable used by the CB Core environment so we run into an unexpected collision here when we allow it to override if present.  I would just rename it, but the BFD server relies on its presence to boot up.  For now I'm just removing the optional override and ensuring we always get the value from file.  Would be good to have @karlmdavis think about this and see if there's any possible scenario this could break local development.

Reply:
Confirmed this breaks some local development cases, refactored to only change behavior when running in Jenkins.